### PR TITLE
[hi] Add content/hi/docs/tasks/tools/included/verify-kubectl.md

### PR DESCRIPTION
**Pull request by vedant-kakde**
Localizes

-  `content/hi/docs/tasks/tools/included/verify-kubectl.md`

### CODE REVIEW

**Comment by vedant-kakde** [comment]
This PR is in continuation of hi localization effort: #29353

/language hi

File: content/hi/docs/tasks/tools/included/verify-kubectl.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: "kubectl इंस्टॉल सत्यापित करें"
+description: "kubectl कैसे सत्यापित करें."
```
Comment (verma-kunal) [inline content/hi/docs/tasks/tools/included/verify-kubectl.md:3]:
```suggestion
description: "kubectl कैसे सत्यापित करें|"
```

Comment (verma-kunal) [reply content/hi/docs/tasks/tools/included/verify-kubectl.md:3]: we use often use `|` or `।` (called `पूर्ण विराम`) in place of a full-stop

Comment (vedant-kakde) [reply content/hi/docs/tasks/tools/included/verify-kubectl.md:3]: Thanks for reviewing Kunal.

File: content/hi/docs/tasks/tools/included/verify-kubectl.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+headless: true
+---
+
+kubectl को कुबेरनेट्स क्लस्टर को खोजने और एक्सेस करने के लिए, उसे
+[kubeconfig file](/docs/concepts/configuration/organize-cluster-access-kubeconfig/) की आवश्यकता होती है,
```
Comment (verma-kunal) [inline content/hi/docs/tasks/tools/included/verify-kubectl.md:8]:
```suggestion
[क्यूबकॉन्फिग फाइल](/docs/concepts/configuration/organize-cluster-access-kubeconfig/)(kubeconfig) की आवश्यकता होती है,
```

Comment (verma-kunal) [reply content/hi/docs/tasks/tools/included/verify-kubectl.md:8]: this makes things more clear! we want readers to know how to pronounce it but we also want them to know this is a `kubernetes object` and how it looks in English

File: content/hi/docs/tasks/tools/included/verify-kubectl.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+जो स्वचालित रूप से तब बनता है जब आप
+[kube-up.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-up.sh) का उपयोग करके क्लस्टर बनाते हैं
+या मिनीक्यूब क्लस्टर को सफलतापूर्वक डेप्लॉय करते हैं.
```
Comment (verma-kunal) [inline content/hi/docs/tasks/tools/included/verify-kubectl.md:11]:
```suggestion
या मिनीक्यूब क्लस्टर को सफलतापूर्वक डिप्लॉय करते हैं.
```

Comment (verma-kunal) [reply content/hi/docs/tasks/tools/included/verify-kubectl.md:11]: i would suggest always refer to the sheet here: https://docs.google.com/spreadsheets/d/1UmL4K0v4yaN34k4WzYDZBobLBp5OZQp4GCTi28AfwQM/edit#gid=0
As many words have been added to that!!

File: content/hi/docs/tasks/tools/included/verify-kubectl.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+डिफ़ॉल्ट रूप से, kubectl कॉन्फ़िगरेशन `~/.kube/config` पर स्थित होता है.
+
+जाँच करें कि क्लस्टर स्टेट प्राप्त करके kubectl ठीक से कॉन्फ़िगर किया गया है:
+
+```shell
+kubectl cluster-info
+```
+
+यदि आप एक URL प्रतिक्रिया देखते हैं, तो kubectl आपके क्लस्टर को एक्सेस करने के लिए सही ढंग से कॉन्फ़िगर किया गया है.
```
Comment (verma-kunal) [inline content/hi/docs/tasks/tools/included/verify-kubectl.md]:
```suggestion
यदि आप एक URL प्रतिक्रिया देखते हैं, तो kubectl आपके क्लस्टर को एक्सेस करने के लिए सही ढंग से कॉन्फ़िगर किया गया है|
```

Comment (verma-kunal) [reply content/hi/docs/tasks/tools/included/verify-kubectl.md]: I'll leave similar changes to you 🙂

File: content/hi/docs/tasks/tools/included/verify-kubectl.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+---
+
+kubectl को कुबेरनेट्स क्लस्टर को खोजने और एक्सेस करने के लिए, उसे
+[kubeconfig file](/docs/concepts/configuration/organize-cluster-access-kubeconfig/) की आवश्यकता होती है,
+जो स्वचालित रूप से तब बनता है जब आप
+[kube-up.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-up.sh) का उपयोग करके क्लस्टर बनाते हैं
+या मिनीक्यूब क्लस्टर को सफलतापूर्वक डेप्लॉय करते हैं.
+डिफ़ॉल्ट रूप से, kubectl कॉन्फ़िगरेशन `~/.kube/config` पर स्थित होता है.
+
+जाँच करें कि क्लस्टर स्टेट प्राप्त करके kubectl ठीक से कॉन्फ़िगर किया गया है:
```
Comment (verma-kunal) [inline content/hi/docs/tasks/tools/included/verify-kubectl.md:14]:
```suggestion
जाँच करें कि क्लस्टर स्टेट प्राप्त करके kubectl को ठीक से कॉन्फ़िगर किया गया है:
```

Comment (verma-kunal) [reply content/hi/docs/tasks/tools/included/verify-kubectl.md:14]: this makes the sentence more clearer!!

File: content/hi/docs/tasks/tools/included/verify-kubectl.md (hunk 4 above)
Comment (verma-kunal) [inline content/hi/docs/tasks/tools/included/verify-kubectl.md]:
```suggestion
यदि आपको  एक URL प्रतिक्रिया दिखती  हैं, तो kubectl आपके क्लस्टर तक पहुँचने के लिए सही ढंग से कॉन्फ़िगर हुआ  है।
```

Comment (verma-kunal) [reply content/hi/docs/tasks/tools/included/verify-kubectl.md]: made some changes here! do share your thoughts, as this looks more appropriate
One tip: while localizing, imagine that you are reading the sentence for 1st time (as a beginner). Think of it as this way & try to figure out what could be the best fit
Overall, the main purpose is to make the reader understand the concepts

File: content/hi/docs/tasks/tools/included/verify-kubectl.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+```shell
+kubectl cluster-info
+```
+
+यदि आप एक URL प्रतिक्रिया देखते हैं, तो kubectl आपके क्लस्टर को एक्सेस करने के लिए सही ढंग से कॉन्फ़िगर किया गया है.
+
+यदि आपको निम्न जैसा संदेश दिखाई देता है, तो kubectl ठीक से कॉन्फ़िगर नहीं किया गया है या कुबेरनेट्स क्लस्टर से कनेक्ट करने में सक्षम नहीं है.
```
Comment (verma-kunal) [inline content/hi/docs/tasks/tools/included/verify-kubectl.md]:
```suggestion
यदि आपको निम्न जैसा संदेश दिखाई देता है, तो kubectl ठीक से कॉन्फ़िगर नहीं हुआ है या कुबेरनेट्स क्लस्टर से कनेक्ट करने में सक्षम नहीं है| 
```

File: content/hi/docs/tasks/tools/included/verify-kubectl.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+
+```
+The connection to the server <server-name:port> was refused - did you specify the right host or port?
+```
+
+उदाहरण के लिए, यदि आप अपने लैपटॉप (स्थानीय रूप से) पर कुबेरनेट्स क्लस्टर चलाने का इरादा रखते हैं, तो आपको पहले मिनीक्यूब जैसे टूल को इंस्टॉल करना होगा और फिर ऊपर बताए गए कमांड को फिर से चलाना होगा।
```
Comment (verma-kunal) [inline content/hi/docs/tasks/tools/included/verify-kubectl.md:28]:
```suggestion
उदाहरण के लिए, यदि आप अपने लैपटॉप (स्थानीय रूप से) पर कुबेरनेट्स क्लस्टर चलाने का इरादा रखते हैं, तो आपको पहले मिनीक्यूब (minikube) जैसे टूल को इंस्टॉल करना होगा और फिर ऊपर बताए गए कमांड को फिर से चलाना होगा।
```

File: content/hi/docs/tasks/tools/included/verify-kubectl.md (hunk 6 above)
Comment (vedant-kakde) [reply content/hi/docs/tasks/tools/included/verify-kubectl.md]: please review updated one

File: content/hi/docs/tasks/tools/included/verify-kubectl.md (hunk 7 above)
Comment (verma-kunal) [reply content/hi/docs/tasks/tools/included/verify-kubectl.md:28]: try mentioning the English names of certain tools/ technologies as well (discussed above as well)

File: content/hi/docs/tasks/tools/included/verify-kubectl.md (hunk 6 above)
Comment (vedant-kakde) [reply content/hi/docs/tasks/tools/included/verify-kubectl.md]: I'm unable to commit these suggested changes as you are reviewing older commit.

File: content/hi/docs/tasks/tools/included/verify-kubectl.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+
+यदि kubectl cluster-info url प्रतिक्रिया देता है, लेकिन आप अपने क्लस्टर को एक्सेस नहीं कर सकते हैं, तो यह जाँचने के लिए कि क्या यह ठीक से कॉन्फ़िगर किया गया है, उपयोग करें:
```
Comment (verma-kunal) [inline content/hi/docs/tasks/tools/included/verify-kubectl.md:30]:
```suggestion
यदि kubectl cluster-info URL प्रतिक्रिया देता है, लेकिन आप अपने क्लस्टर को एक्सेस नहीं कर सकते हैं, तो यह जाँचने के लिए कि क्या यह ठीक से कॉन्फ़िगर किया गया है, उपयोग करें:
```

Comment (vedant-kakde) [reply content/hi/docs/tasks/tools/included/verify-kubectl.md:30]: Its is in small-case in en doc

**Review by verma-kunal — COMMENTED** [review]
LGTM 👍

**Comment by anubha-v-ardhan** [comment]
/retitle [hi] Add content/hi/docs/tasks/tools/included/verify-kubectl.md

File: content/hi/docs/tasks/tools/included/verify-kubectl.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+या मिनीक्यूब क्लस्टर को सफलतापूर्वक डिप्लॉय करते हैं.
+डिफ़ॉल्ट रूप से, kubectl कॉन्फ़िगरेशन `~/.kube/config` पर स्थित होता है|
+
+जाँच करें कि क्लस्टर स्टेट प्राप्त करके kubectl को ठीक से कॉन्फ़िगर किया गया है:
+
+```shell
+kubectl cluster-info
+```
+
+यदि आपको एक URL प्रतिक्रिया दिखती  हैं, तो kubectl आपके क्लस्टर तक पहुँचने के लिए सही ढंग से कॉन्फ़िगर हुआ है।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/verify-kubectl.md:20]:
```suggestion
यदि आपको एक URL प्रतिक्रिया दिखती हैं, तो kubectl आपके क्लस्टर तक पहुँचने के लिए सही ढंग से कॉन्फ़िगर हुआ है।
```

File: content/hi/docs/tasks/tools/included/verify-kubectl.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+यदि आपको निम्न जैसा संदेश दिखाई देता है, तो kubectl ठीक से कॉन्फ़िगर नहीं हुआ है या कुबेरनेट्स क्लस्टर से कनेक्ट करने में सक्षम नहीं है|
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/verify-kubectl.md:22]:
```suggestion
यदि आपको निम्नलिखित संदेश दिखाई देता है, तो kubectl ठीक से कॉन्फ़िगर नहीं हुआ है या कुबेरनेट्स क्लस्टर से कनेक्ट करने में सक्षम नहीं है|
```

File: content/hi/docs/tasks/tools/included/verify-kubectl.md [hunk 11]
```diff
@@ -0,0 +1,34 @@
...
+
+```
+The connection to the server <server-name:port> was refused - did you specify the right host or port?
+```
+
+उदाहरण के लिए, यदि आप अपने लैपटॉप (स्थानीय रूप से) पर कुबेरनेट्स क्लस्टर चलाने का इरादा रखते हैं, तो आपको पहले मिनीक्यूब (minikube) जैसे टूल को इंस्टॉल करना होगा और फिर ऊपर बताए गए कमांड को फिर से चलाना होगा।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/verify-kubectl.md:28]:
```suggestion
उदाहरण के लिए, यदि आप अपने लैपटॉप (स्थानीय रूप से) पर कुबेरनेट्स क्लस्टर चलाना चाहते हैं, तो आपको पहले मिनीक्यूब (minikube) जैसे टूल को इंस्टॉल करना होगा और ऊपर बताए गए कमांड को फिर से चलाना होगा।
```

File: content/hi/docs/tasks/tools/included/verify-kubectl.md [hunk 12]
```diff
@@ -0,0 +1,34 @@
...
+
+यदि kubectl cluster-info URL प्रतिक्रिया देता है, लेकिन आप अपने क्लस्टर को एक्सेस नहीं कर पा रहें हैं, तो यह जाँचने के लिए कि क्या यह ठीक से कॉन्फ़िगर किया गया है, निचे दी गयी कमांड का उपयोग करें:
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/verify-kubectl.md:30]:
```suggestion
यदि kubectl cluster-info URL प्रतिक्रिया देता है, लेकिन आप अपने क्लस्टर को एक्सेस नहीं कर पा रहें हैं, तो यह जाँचने के लिए कि क्या यह ठीक से कॉन्फ़िगर किया गया है, इस कमांड का उपयोग करें:
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Final Review
If you agree with the changes, commit these and then squash all the commits into one commit.
This should make this PR ready to merge.

**Comment by vedant-kakde** [comment]
@anubha-v-ardhan Done ✅

**Comment by anubha-v-ardhan** [comment]
@vedant-kakde Thank you. Can you squash all the commits into one commit.

**Comment by vedant-kakde** [comment]
@anubha-v-ardhan Done ✅

**Comment by anubha-v-ardhan** [comment]
@vedant-kakde looks like you mistakenly squashed commits that were not related to this PR as there are 3 file changes showing up.

**Comment by vedant-kakde** [comment]
Those commits were done by maintainers I guess

**Comment by vedant-kakde** [comment]
So I squashed them as there are some valid changes

**Comment by vedant-kakde** [comment]
Anything else I can do here ?

**Comment by anubha-v-ardhan** [comment]
@vedant-kakde Did you rebase against local `main` instead of `dev-1.22-hi.1`. Please refer [This thread on Slack](https://kubernetes.slack.com/archives/C1J0BPD2M/p1631535983093300) and implement either of the two solution proposed by @sftim or make a complete new fresh PR. Whichever is easier for you.

**Comment by vedant-kakde** [comment]
As you can see I already squashed the extra commits into single commit but some unwanted commits are merged in my PR.

**Comment by anubha-v-ardhan** [comment]
Yes, that is why, follow the instructions in above comment :)

File: content/hi/docs/tasks/tools/included/verify-kubectl.md [hunk 13]
```diff
@@ -0,0 +1,34 @@
+---
+title: "kubectl इंस्टॉल सत्यापित करें"
+description: "kubectl कैसे सत्यापित करें|"
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/verify-kubectl.md:3]:
```suggestion
description: "kubectl कैसे सत्यापित करें।"
```

Comment (anubha-v-ardhan) [reply content/hi/docs/tasks/tools/included/verify-kubectl.md:3]: Please note that `|` and `।` are different characters. We should prefer `।` for पूर्ण विराम

**Comment by vedant-kakde** [comment]
@anubha-v-ardhan Done ✅

File: content/hi/docs/tasks/tools/included/verify-kubectl.md [hunk 14]
```diff
@@ -0,0 +1,34 @@
...
+headless: true
+---
+
+kubectl को कुबेरनेट्स क्लस्टर को खोजने और एक्सेस करने के लिए, उसे
+[क्यूबकॉन्फिग फाइल](/docs/concepts/configuration/organize-cluster-access-kubeconfig/)(kubeconfig) की आवश्यकता होती है,
+जो स्वचालित रूप से तब बनता है जब आप
+[kube-up.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-up.sh) का उपयोग करके क्लस्टर बनाते हैं
+या मिनीक्यूब क्लस्टर को सफलतापूर्वक डिप्लॉय करते हैं.
+डिफ़ॉल्ट रूप से, kubectl कॉन्फ़िगरेशन `~/.kube/config` पर स्थित होता है|
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/verify-kubectl.md:12]:
```suggestion
[kube-up.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/kube-up.sh) का उपयोग करके क्लस्टर बनाते हैं
या मिनीक्यूब क्लस्टर को सफलतापूर्वक डिप्लॉय करते हैं।
डिफ़ॉल्ट रूप से, kubectl कॉन्फ़िगरेशन `~/.kube/config` पर स्थित होता है।
```

File: content/hi/docs/tasks/tools/included/verify-kubectl.md [hunk 15]
```diff
@@ -0,0 +1,34 @@
...
+
+जाँच करें कि क्लस्टर स्टेट प्राप्त करके kubectl को ठीक से कॉन्फ़िगर किया गया है:
+
+```shell
+kubectl cluster-info
+```
+
+यदि आपको एक URL प्रतिक्रिया दिखती हैं, तो kubectl आपके क्लस्टर तक पहुँचने के लिए सही ढंग से कॉन्फ़िगर हुआ है।
+
+यदि आपको निम्नलिखित संदेश दिखाई देता है, तो kubectl ठीक से कॉन्फ़िगर नहीं हुआ है या कुबेरनेट्स क्लस्टर से कनेक्ट करने में सक्षम नहीं है|
```
Comment (anubha-v-ardhan) [inline content/hi/docs/tasks/tools/included/verify-kubectl.md:22]:
```suggestion
यदि आपको निम्नलिखित संदेश दिखाई देता है, तो kubectl ठीक से कॉन्फ़िगर नहीं हुआ है या कुबेरनेट्स क्लस्टर से कनेक्ट करने में सक्षम नहीं है।
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Comment by anubha-v-ardhan** [comment]
@vedant-kakde Commit the above suggestions and we should be ok to merge this PR.

**Pull request merged**